FoodOrderingSystem.py

```diff
@@ -0,0 +1,925 @@
+
+    def delete_order(self, order_id):
+        session = self.Session()
+        self.employee.delete_order(order_id, session)
+        print("Order {} deleted!".format(order_id))
+
+    def view_menu(self):
+        session = self.Session()
+        menu = self.customer.view_menu(session)
+        for fc, fd in menu:
+            print("\nFood ID: {} \nFood category: {} \nFood name: {} \nFood price: {}".format(fd.food_id, fc.name, fd.food_name, fd.price))
+
+    def customer_signup(self, cust_name, cust_phone, cust_email):
+        session = self.Session()
+        c = self.customer.customer_signup(cust_name, cust_phone, cust_email, session)
+        if c:
+            print("\nCustomer ID: {} \nCustomer name: {} \nCustomer phone number: {} \nCustomer email address: {} \nSucessfully!".format(c.cust_id, c.cust_name, c.cust_phone, c.cust_email))
+        else:
+            print("Adding details was not sucessfully. Please try again!")    
+
+    def customer_login(self, cust_id):
+        session = self.Session()
+        login = self.customer.customer_login(cust_id, session)
+        return login
+
+    def process_order(self, cust_id):
+        session = self.Session()
+        o = self.customer.create_order_id(cust_id, session)
+        selection = """
+        
+        0. Back
+        1. Add food to order
+        2. Remove food to order
+        3. Update food to order
+                
+        Select option: 
+
+        """
+        option = int(input(selection))
+
+        while option != 0:
+            
+            if option == CUST_OPT_ADD_FOOD_TO_ORDER:
+                food_id = input("Enter food ID: ")
+                food_qty = input("Enter food quantity: ")
+                add = self.customer.add_food_to_order(o.order_id, food_id, food_qty, session)
+                if add:
+                    print("Add successful!")
+                else:
+                    print("Add not successful. Please try again!")
+
+            elif option == CUST_OPT_REMOVE_FOOD_TO_ORDER:
+                food_id = input("Enter food ID: ")
+                remove = self.customer.remove_food_to_order(o.order_id, food_id, session)
+                if remove:
+                    print("Items removed")
+                else:
+                    print("Items not removed")
+
+            elif option == CUST_OPT_UPDATE_FOOD_TO_ORDER:
+                food_id = input("Enter food ID: ")
+                food_qty = input("Enter food quantity: ")
+                update = self.customer.update_food_to_order(o.order_id, food_id, food_qty, session)
+                if update:
+                    print("Items updated")
+                else:
+                    print("Items not updated")
+
+            option = int(input(selection))
+        
+        if o:
+            print("\nYour order is generated. \nOrder number: {}".format(o.order_id))
+        else:
+            print("Order not generated. Please try again!")
```

Comment:
This check can be made above in line 584:

```python
o = self.customer.create_order_id(cust_id, session)
if not o:
  print("Order not generated. Please try again!")
  return

selection = """
        
        0. Back
        1. Add food to order
        2. Remove food to order
        3. Update food to order
                
        Select option: 
        """
option = int(input(selection))

...
...

```

Reply:
Because if the order is not generated in the first place, there's no point in add/remove from from the order.